aimmdb/ingest/eli.py

```diff
@@ -37,6 +37,13 @@ def read_metadata_and_header(path):
     return metadata, header
 
 
+def add_tiled_uid(md: dict):
```

Comment:
One thing I learned from Juan recently is that Tiled will actually do this for us. We never need to add uids to the metadata ourselves! No way you could've known but please feel free to remove this functionality.

Reply:
(Btw sorry I didn't mention this earlier, I only found out relatively recently)